.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,3 @@
-<!-- YOU SHOULD KNOW: dotcom is running an experiment called PR Deployments. -->
```

Comment:
Might be useful to leave a message here to explain how to get a live link using the label?

Reply:
Can do!